app/src/main/java/xyz/jhughes/laundry/LaundryParser/MachineStates.java

```diff
@@ -0,0 +1,26 @@
+package xyz.jhughes.laundry.LaundryParser;
+
+/**
+ * Created by dmtsc on 10/18/2016.
+ */
+
+public class MachineStates {
+    public static final String IN_USE = "In use";
```

Comment:
Should probably move these into strings.xml, but it works for now so we can get a release out.